Common/cpp/AnimatedSensor/AnimatedSensorModule.cpp

```diff
@@ -33,7 +29,9 @@ jsi::Value AnimatedSensorModule::registerSensor(
   auto uiRuntime = runtimeHelper->uiRuntime();
 
   int sensorId = platformRegisterSensorFunction_(
-      sensorType, interval.asNumber(), [=](double newValues[]) {
+      sensorType,
+      interval.asNumber(),
+      [uiRuntime, sensorType, shareableHandler](double newValues[]) {
         jsi::Runtime &rt = *uiRuntime;
```

Comment:
Is it really safe to access `uiRuntime` here?

If not, we can capture `runtimeHelper` (which is a shared pointer) to the lambda and call `runtimeHelper->uiRuntime();` inside (like we do in many other places).

Reply:
I think this is outside the scope of this PR, but I might as well. 👍